<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiplyDivide_game</class>
 <widget class="QMainWindow" name="MultiplyDivide_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background-color: none;
font: 15pt&quot;BM Jua&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <widget class="QLabel" name="scoreUpdate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: none;
font: 20pt &quot;BM Jua&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="divisor">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: black;
font: 48pt &quot;BM Jua&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="doneButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: black;</string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionUpdate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
font: 20pt &quot;BM Jua&quot;;
color: black;</string>
    </property>
    <property name="text">
     <string>1 / 10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/image for c++ project/Background/quickMathBackground.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="operandSymbol">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: red;
font: 65pt &quot;Baloo 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="augend">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: black;
font: 48pt &quot;BM Jua&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="userInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
background-color: rgb(255, 252, 236);
font: 48pt &quot;BM Jua&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
border: none;
border-width: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/image for c++ project/Button/exit.png</normaloff>:/image for c++ project/Button/exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>scoreUpdate</zorder>
   <zorder>divisor</zorder>
   <zorder>doneButton</zorder>
   <zorder>questionUpdate</zorder>
   <zorder>operandSymbol</zorder>
   <zorder>augend</zorder>
   <zorder>userInput</zorder>
   <zorder>exitButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
